spec/support/features/post_helpers.rb

```diff
@@ -0,0 +1,18 @@
+module Features
+  module PostHelpers
+    def create_post(title, url)
+      visit(new_post_path)
+      fill_in('Title', with: title)
+      fill_in('Url', with: url)
+      click_button('Create Post')
+    end
+  end
+end
+
+RSpec.configure do |config|
+  config.include Features::PostHelpers, type: :feature
+end
```

Comment:
Ahh, cool. Yeah I'd already gotten like 3-4 of those config.include blocks for different helpers, so it's certainly clearer to have them in one place. Thanks.